src/python/tensorflow_cloud/core/run.py

```diff
@@ -225,9 +226,11 @@ def run(
 
     # Delete all the temporary files we created.
     if preprocessed_entry_point is not None:
+        os.close(pep_file_descriptor)
         os.remove(preprocessed_entry_point)
-    for f in container_builder.get_generated_files():
-        os.remove(f)
+    for file_path, file_descriptor in container_builder.get_generated_files(return_descriptors=True):
```

Comment:
Can you format this so that line length <= 80?

Reply:
https://github.com/tensorflow/cloud/pull/218/commits/03dd09546460757924104957abb43d5dd1623a89